includes/class-coblocks-block-assets.php

```diff
@@ -352,17 +352,6 @@ public function frontend_scripts() {
 			true
 		);
 
-		// Masonry block.
-		if ( $this->is_page_gutenberg() || has_block( 'coblocks/gallery-masonry' ) || has_block( 'core/block' ) ) {
-			wp_enqueue_script(
-				'coblocks-masonry',
-				$dir . 'coblocks-masonry.js',
-				array( 'jquery', 'masonry', 'imagesloaded' ),
-				COBLOCKS_VERSION,
-				true
-			);
-		}
```

Comment:
I think we still need this because the v2 Masonry block is not compatible with WordPress version below 5.7.2. We allow users to use both versions of the block using a special wrapper and we need to keep the old logic around for that purpose.

Reply:
No, we are good on this I suppose. Turns out the Masonry block broke with 5.6 somewhere along the way. Let us go ahead and get this in and then we will have to look at that Masonry block later.